omero/users/history.rst

```diff
@@ -1,7 +1,3 @@
-.. Content for this page should be opened as a PR against
```

Comment:
I forgot to add this banner to the page in the code repo, I'll open a quick PR now to do that